packages/local_auth/local_auth_ios/test/local_auth_test.dart

```diff
@@ -0,0 +1,112 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:local_auth_ios/local_auth_ios.dart';
+import 'package:local_auth_platform_interface/types/auth_options.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  group('LocalAuth', () {
+    const MethodChannel channel = MethodChannel(
+      'plugins.flutter.io/local_auth_ios',
+    );
+
+    final List<MethodCall> log = <MethodCall>[];
+    late LocalAuthIOS localAuthentication;
+
+    setUp(() {
+      channel.setMockMethodCallHandler((MethodCall methodCall) {
+        log.add(methodCall);
+        return Future<dynamic>.value(true);
+      });
+      localAuthentication = LocalAuthIOS();
+      log.clear();
+    });
+
```

Comment:
Looks like we're missing simple tests for the other methods; they should definitely be added.